src/LinearAlgebra/helper.jl

```diff
@@ -694,7 +694,7 @@ struct BlockInterlacer{DMS<:Tuple}
 end
 
 
-const TrivialInterlacer{d} = BlockInterlacer{NTuple{d,<:Ones}}
+const TrivialInterlacer{d} = BlockInterlacer{<:NTuple{d,<:Ones}}
```

Comment:
Perhaps this can just be `<:NTuple{d,Ones}`, as 
```julia
julia> (NTuple{d,<:T} where {d,T}) == (NTuple{d,T} where {d,T})
true
```

Reply:
changed